### PR TITLE
a request to add my plugin to flow launcher

### PR DESCRIPTION
**Pull request by DevHJS**
a url shortener that use tiny url api to shorten links

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -2133,5 +2133,20 @@
...
+        "Name": "TinyUrlPlugin",
+        "Description": "a plugin that use tinyurl api, it's very basic but powerful",
+        "Author": "haider",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/DevHJS/TinyUrlPlugin",
+        "UrlDownload": "https://github.com/DevHJS/TinyUrlPlugin/releases/download/v1.0.0/tiny.url.plugin.zip",
+        "UrlSourceCode": "https://github.com/DevHJS/TinyUrlPlugin/tree/master",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/DevHJS/TinyUrlPlugin/Flow.Launcher.Plugin.TinyUrlPlugin/icons/icon.png",
+        "Tested": true
```
Comment (jjw24) [inline plugins.json:2149]:
```suggestion
```

**Comment by DevHJS** [comment]
I added the file like you asked, not gonna lie, I never used github actions and yaml before
https://github.com/DevHJS/TinyUrlPlugin/blob/master/Flow.Launcher.Plugin.TinyUrlPlugin/.github/workflows/publish-action.yaml
Can you review it please?

**Comment by jjw24** [comment]
> I added the file like you asked, not gonna lie, I never used github actions and yaml before https://github.com/DevHJS/TinyUrlPlugin/blob/master/Flow.Launcher.Plugin.TinyUrlPlugin/.github/workflows/publish-action.yaml Can you review it please?

So GitHub action is used to automatically do a release of your plugin when you bump the version in your plugin.json. that `.github/workflows folder should be placed on the top level of your repo in order for the Action to run. Once done you can just update your plugin.json's version to something like 1.0.1 and the action should run and create a release.

For reasons we don't want plugin devs to upload self-created releases.

**Comment by DevHJS** [comment]
So after multiple failures i succeeded in making a functional github release action...
https://github.com/DevHJS/TinyUrlPlugin/releases
https://github.com/DevHJS/TinyUrlPlugin/blob/master/.github/workflows/publish.yml
i was busy with my work and family so it took more time than it should

File: plugins.json [hunk 2]
```diff
@@ -2161,5 +2161,17 @@
...
+    },
+    {
+        "ID": "589CA535BD5041E78179C4F2D2F2D8FD",
+        "Name": "TinyUrlPlugin",
+        "Description": "a plugin that use tinyurl api, it's very basic but powerful",
+        "Author": "haider",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/DevHJS/TinyUrlPlugin",
+        "UrlDownload": "https://github.com/DevHJS/TinyUrlPlugin/releases/download/v1.0.0/tiny.url.plugin.zip",
```
Comment (jjw24) [inline plugins.json:2173]:
```suggestion
        "UrlDownload": "https://github.com/DevHJS/TinyUrlPlugin/releases/download/v1.0.5/tiny.url.plugin.zip",
```

File: plugins.json [hunk 3]
```diff
@@ -2161,5 +2161,17 @@
         "IcoPath": "https://cdn.jsdelivr.net/gh/AminSallah/Flow.Launcher.Plugin.WinHotkey@main/app.png",
         "DateAdded": "2024-02-22T06:54:14Z",
         "LatestReleaseDate": "2024-02-20T05:05:01Z"
+    },
+    {
+        "ID": "589CA535BD5041E78179C4F2D2F2D8FD",
+        "Name": "TinyUrlPlugin",
+        "Description": "a plugin that use tinyurl api, it's very basic but powerful",
+        "Author": "haider",
+        "Version": "1.0.0",
```
Comment (jjw24) [inline plugins.json:2170]:
```suggestion
        "Version": "1.0.5",
```

**Review by jjw24 — DISMISSED** [review]

File: plugins.json [hunk 4]
```diff
@@ -2161,5 +2161,17 @@
...
+    },
+    {
+        "ID": "589CA535BD5041E78179C4F2D2F2D8FD",
+        "Name": "TinyUrlPlugin",
+        "Description": "a plugin that use tinyurl api, it's very basic but powerful",
+        "Author": "haider",
+        "Version": "1.0.5",
+        "Language": "csharp",
+        "Website": "https://github.com/DevHJS/TinyUrlPlugin",
+        "UrlDownload": "https://github.com/DevHJS/TinyUrlPlugin/releases/download/v1.0.5/tiny.url.plugin.zip",
```
Comment (jjw24) [inline plugins.json:2173]:
```suggestion
        "UrlDownload": "https://github.com/DevHJS/TinyUrlPlugin/releases/download/v1.0.5/TinyUrlPlugin-1.0.5.zip",
```

**Review by jjw24 — APPROVED** [review]

**Pull request merged**